### PR TITLE
Update Program.cs

### PR DESCRIPTION
**Pull request by bertt**
. removed unused usings
. make function Main static

### CODE REVIEW

File: templates/projects/console/Program.cs [hunk 1]
```diff
@@ -1,13 +1,10 @@
 using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Threading.Tasks;
```
Comment (peterblazejewicz) [inline templates/projects/console/Program.cs]: Why the `using` should be removed? As I understand there is a reason behind most common imports being used in blank templates:
https://github.com/aspnet/Templates/blob/dev/src/BaseTemplates/ConsoleApp/Program.cs

File: templates/projects/console/Program.cs [hunk 2]
```diff
@@ -1,13 +1,10 @@
...
 
 namespace <%= namespace %>
 {
     public class Program
     {
-        public void Main(string[] args)
+        public static void Main(string[] args)
```
Comment (peterblazejewicz) [inline templates/projects/console/Program.cs]: LGTM
https://github.com/aspnet/Templates/commit/d213d9c01c91354bd44af92c270b438c6d2f51df#diff-05e5de986ba4aba48a0b5a183df88cb8

**Comment by peterblazejewicz** [comment]
@bertt 
Any comment?

File: templates/projects/console/Program.cs (hunk 1 above)
Comment (sayedihashimi) [inline templates/projects/console/Program.cs]: Right, they are there because they are commonly used. Including exposing extension methods.

**Comment by peterblazejewicz** [comment]
@sayedihashimi 
I'll merge this and submit reverting PR for removed `using`. We will have another contributor :)

**Comment by peterblazejewicz** [comment]
The PR was started from wrong branch, something we have missed. It was merged into `beta8` branch, diverging it from current state.
I've repeated aspnet/Templates#240 change in new #456 PR

**Comment by bertt** [comment]
ok thanks!

**Pull request merged**